src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -283,6 +285,64 @@ def update_app_service_plan(instance, sku=None, number_of_workers=None,
         instance.admin_site_name = admin_site_name
     return instance
 
+def show_backup_configuration(resource_group_name, name, slot=None):
+    return _generic_site_operation(resource_group_name, name, 'get_site_backup_configuration',
+                                   slot)
+
+def list_backups(resource_group_name, name, slot=None):
+    return _generic_site_operation(resource_group_name, name, 'list_site_backups',
+                                   slot)
+
+def create_backup(resource_group_name, name, storage_account_url,
+                  database_name=None, database_type=None,
+                  database_connection_string=None, backup_name=None, slot=None):
+    client = web_client_factory()
+    location = _get_location_from_webapp(client, resource_group_name, name)
+    db_setting = DatabaseBackupSetting(database_type, database_name,
+                                       connection_string=database_connection_string)
+    backup_request = BackupRequest(location, backup_request_name=backup_name,
+                                   storage_account_url=storage_account_url, databases=[db_setting])
+    if slot is None:
+        return client.sites.backup_site(resource_group_name, name, backup_request)
+    else:
+        return client.sites.backup_site_slot(resource_group_name, name, backup_request, slot)
+
+def update_backup_schedule(resource_group_name, name, storage_account_url,
+                           schedule_frequency, schedule_frequency_unit,
+                           keep_at_least_one_backup=None,
+                           retention_period_in_days=90, database_name=None,
+                           database_connection_string=None, database_type=None, slot=None):
+    client = web_client_factory()
+    location = _get_location_from_webapp(client, resource_group_name, name)
+    db_setting = DatabaseBackupSetting(database_type, database_name,
+                                       connection_string=database_connection_string)
+    backup_schedule = BackupSchedule(schedule_frequency_unit, schedule_frequency,
+                                     keep_at_least_one_backup, retention_period_in_days)
```

Comment:
As an update, all parameters (except resource_group_name and name...which shouldn't be called name) should have default values of None.  Then you selectively go through and update the properties that are not None. Right now you have an update that works like a PUT (replacing things wholesale) instead of the PATCH-like behavior of a typical update.